### PR TITLE
Fixes Coniine making you suffocate forever

### LINKED ISSUE #59013: Coniine is broken
Reporting client version: 513.1536

## Round ID:
(Don't have it with me)

## Testmerges:
- [Removes bad return from processables that broke cooking ðŸ€](https://www.github.com/tgstation/tgstation/pull/58983)

## Reproduction:
Coniine seems to cause anyone that has it to suffocate forever, even after it gets out of their system.

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Closes #59013 

## Why It's Good For The Game

Wiki says that it **refreshes** 5 ticks of suffocation each tick, while in reality it **adds** 5 ticks of suffocation each tick, stacking it to huge values, making people suffocate forever even after it leaves their system.

## Changelog
:cl:
fix: Coniine no longer makes you suffocate even after it leaves your system
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -842,7 +842,7 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/toxin/coniine/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	M.losebreath += 5 * REM * delta_time
+	M.losebreath = min(M.losebreath + 5 * REM * delta_time, 5)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:845]: we need to make sure we aren't indirectly healing people with more than 5 losebreath prior to having this drug in their system.

Comment (coiax) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm:845]: what about
```suggestion
        var/amount= 5 * REM * delta_time
        if(M.losebreath < amount)
            M.losebreath += amount
```

**Comment by Krysonism** [comment]
This was not a fix.

**Comment by Iamgoofball** [comment]
This was a fuckup on the wiki's part, not the code. It's actually supposed to add +5 losebreath every time it processes, this is consistent with 2016 gooncode, my own notes on the goonchem addition PR, and the old goon wiki.

**Pull request merged**